### ISSUE TITLE
JSON output status always skipped in features with only Outlines

### ISSUE THREAD

**Issue by remcowesterhoud** [behave/behave#442]
When I make the test output results to a JSON file the features that only contains scenario outlines are always marked as skipped. Instead of the actual status (passed/failed).
```
Feature: Example scenario

  Scenario Outline: Addition
    Given I have the number <number1>
    When I add the number <number2>
    Then The result should be <number3>

    Examples:
      | number1 | number2 | number3 |
      | 1       | 1       | 2       |
      | 1       | 2       | 3       |
      | 2       | 3       | 5       |
```
Running this with behave -f json -o test.json -i example.feature --junit results in a test.json file containing 1 feature with a status of skipped. The status in the junit test report passed as expected.
```
Feature: Example scenario

  Scenario Outline: Addition
    Given I have the number <number1>
    When I add the number <number2>
    Then The result should be <number3>

    Examples:
      | number1 | number2 | number3 |
      | 1       | 1       | 2       |
      | 1       | 2       | 3       |
      | 2       | 3       | 5       |

  Scenario: Addition 2
  	Given I have the number 3
  	When I add the number 5
  	Then The result should be 8
```
Running this using the same command results in a test.json file with a status of passed.

I have attached the example feature and steps I've used.
[example.zip](https://github.com/behave/behave/files/240389/example.zip)

**Comment by remcowesterhoud** [comment]
This may not have to do with the json formatter in particular. I just noticed that the summary at the end of the testrun also says skipped even though the scenarios all passed.

0 features passed, 0 failed, 1 skipped
3 scenarios passed, 0 failed, 0 skipped
9 steps passed, 0 failed, 0 skipped, 0 undefined

**Comment by remcowesterhoud** [comment]
Whilst searching why this happened I came across this. I'm not sure that this is the reason it happens but it does seem odd to me.

```py
def compute_status(self):
        """
        Compute the status of this feature based on its:
           * scenarios
           * scenario outlines

        :return: Computed status (as string-enum).
        """
        skipped = True
        passed_count = 0
        for scenario in self.scenarios:
            scenario_status = scenario.status
            if scenario_status == 'failed':
                return 'failed'
            elif scenario_status == 'untested':
                if passed_count > 0:
                    return 'failed'  # ABORTED: Some passed, now untested.
                return 'untested'
            if scenario_status != 'skipped':
                skipped = False
            if scenario_status == 'passed':
                passed_count += 1
        return skipped and 'skipped' or 'passed'
```
If my (limited) knowledge of Python is correct this return will always return True/False and 'skipped'. There is no way it can return 'passed' since a string is always considered to be True.

**Comment by remcowesterhoud** [comment]
Figured out what was wrong. Also found out it was already fixed in f1d561e720ba217e0ebb13e4f70dd94e6a392ec6
